core/domain/topic_services.py

```diff
@@ -267,8 +271,21 @@ def apply_change_list(topic_id, change_list):
                       topic_domain.TOPIC_PROPERTY_LANGUAGE_CODE):
                     topic.update_language_code(change.new_value)
                 elif (change.property_name ==
-                      topic_domain.TOPIC_PROPERTY_THUMBNAIL_FILENAME):
-                    topic.update_thumbnail_filename(change.new_value)
+                      topic_domain.TOPIC_PROPERTY_THUMBNAIL_FILENAME_AND_SIZE_IN_BYTES): # pylint: disable=line-too-long
+                    try:
+                        fs = fs_domain.AbstractFileSystem(
+                            fs_domain.GcsFileSystem(
+                                feconf.ENTITY_TYPE_TOPIC, topic_id))
+                        thumbnail_size_in_bytes = len(fs.get(
+                            '%s/%s' % (
+                                ASSET_TYPE_THUMBNAIL,
```

Comment:
Sean's inputs:
Check how the thumbnail is saved, use the same logic for retrieval as well.

Reply:
Done